scripts/maintenance/migrate_project/Makefile

```diff
@@ -1,23 +1,36 @@
-.DEFAULT_GOAL := build
+.DEFAULT_GOAL := help
 
 SHELL := /bin/bash
 
-CFG_NAME	:= cfg.ignore.json
+CFG_NAME	:= cfg.json
 REPO_DIR	:= $(CURDIR)/../../..
 IMAGE_NAME	:= migrate-utils
 TAG			:= latest
 
-build:
+.PHONY: help
+
+help: ## help on rule's targets
+ifeq ($(IS_WIN),)
+	@awk --posix 'BEGIN {FS = ":.*?## "} /^[[:alpha:][:space:]_-]+:.*?## / {printf "\033[36m%-20s\033[0m %s\n", $$1, $$2}' $(MAKEFILE_LIST)
+else
+	@awk --posix 'BEGIN {FS = ":.*?## "} /^[[:alpha:][:space:]_-]+:.*?## / {printf "%-20s %s\n", $$1, $$2}' $(MAKEFILE_LIST)
+endif
+
+.PHONY: build
+build: ## Builds docker container for the migration. Run this first.
 	docker build -t ${IMAGE_NAME} --file $(PWD)/Dockerfile $(REPO_DIR)
 
-shell:
+.PHONY: shell
+debug-shell: ## Runs a bash inside the container that performs the migrate. For manual interventions and debuging.
 	docker run -it --rm ${IMAGE_NAME}:${TAG} bash
 
-empty-config-file:	# empty configuration file to fill up
+.PHONY: create-empty-config-file
+create-empty-config-file:	## creates an empty configuration file to fill up
```

Comment:
``Makefile`` was created in the old times (you were not born :-) ) with the idea of "making a given file"

For instance, to create ``cfg.json``, you would do directly ``make cfg.json``

If ``cfg.json`` is not there or the timestamp is older than the dependencies, it would make it again ... and so on.
If you want ALWAYS to build it, regardless of its age, then you do
```Makefile
.PHONY: cfg.json
cfg.json:
     docker run -it --rm \
	${IMAGE_NAME}:${TAG} python models.py > $@
```

using verbs (e.g. ``run``, ``build`` ... ) instead of file as targets is something that you use when there is no real file to refer to ...